### PR TITLE
Long/Close didn't load a user's oSQTH balance

### PR DESCRIPTION
**Pull request by EstherSeyi**
# Task:

## Description

- Adds activity update for when sqthBal is loading or when there is an error loading the sqth bal.
- Allow user to refetch incase sqth bal fails to load. 

Fixes # (issue)
#220 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

### CODE REVIEW

**Review by yichiehliu — COMMENTED** [review]
is it possible to make long squeeth balance an atom? cuz the oSQTH balance in trade card is still not updated after a close trade on long side, still required a refresh or click on other tabs to get the correct balance

**Review by yichiehliu — CHANGES_REQUESTED** [review]
Were you able to test with long close? cuz the tx would fail for me

Uploading 螢幕錄製 2022-04-29 上午5.05.23.mov…

**Comment by yichiehliu** [comment]
ok, the tx succeeded after several hard refreshes, not sure if it's related to the changes

File: packages/frontend/src/components/Trade/Long/index.tsx [hunk 1]
```diff
@@ -960,11 +985,31 @@ type BuyProps = {
...
 const Long: React.FC<BuyProps> = ({ open, isLPage = false, activeStep = 0 }) => {
+  const { longSqthBal, refetch: refetchSqthBal, loading: loadingSqthBal, error } = useLongSqthBal()
+
   return open ? (
-    <OpenLong open={open} isLPage={isLPage} activeStep={activeStep} />
+    <OpenLong
+      open={open}
+      isLPage={isLPage}
+      activeStep={activeStep}
+      sqthBalProps={{ longSqthBal, refetch: refetchSqthBal, loading: loadingSqthBal }}
```
Comment (yichiehliu) [inline packages/frontend/src/components/Trade/Long/index.tsx:1005]: I noticed there are some inconsistencies for long balance in the codebase, cuz there is also long balance in the osqth input of <OpenLong /> component, but the balance will change right after the open long trade finished, it gets updated with the same value on position card and at the same time. 

We dont have balance not loaded issue on buy side, cuz we r using [squeethAmount variable ](https://github.com/opynfinance/squeeth-monorepo/pull/385/files#diff-0df6f866c3e8aa6fd371a3fdb870c356e099f912847729a6c7693c57597901fcR511) as our long balance there, same as the value on position card and position page, but on long close, now we r using useLongSqthBal, which does some extra fetching and this hook is only used in long trade card, but think squeethAmount already did the fetching job to get correct position value. So we dont need extra fetching in long close component

So thinking it's better to have a jotai atom for long position value across the app, or at least we all use same value / variable for same position

**Review by yichiehliu — CHANGES_REQUESTED** [review]

**Review by yichiehliu — APPROVED** [review]

**Review by adam5909099 — APPROVED** [review]

**Review by KMKoushik — APPROVED** [review]

**Pull request merged**